lib/kalevala/character/conn.ex

```diff
@@ -213,11 +213,33 @@ defmodule Kalevala.Character.Conn do
   """
   def get_session(conn, key), do: Map.get(conn.session, key)
 
+  @doc """
+  Put a value in to the flash data. Flash data is reset every
+  time the controller is switched.
+  """
+  def put_flash(conn, key, value) do
```

Comment:
I think I'd rather have `put_flash/3` and `get_flash/2` use a separate map so they can be set and cleared separately from the session.

`put_controller/2` below should be good to reset so how it was and then in https://github.com/oestrich/kalevala/blob/main/lib/kalevala/character/foreman.ex#L254 you can add `flash: %{}` to have it reset